aten/src/ATen/quantized/Quantizer.cpp

```diff
@@ -104,18 +104,18 @@ Tensor PerTensorAffineQuantizer::quantize(Tensor rtensor) {
   // quantizer that can be reused, so I'm using intrusive_from_this here
   Tensor qtensor = new_qtensor(
       rtensor.sizes(),
-      rtensor.options().dtype(scalar_type_),
+      rtensor.options().dtype(scalar_type_).memory_format(rtensor.suggest_memory_format()),
       intrusive_from_this());
 
-  rtensor = rtensor.contiguous();
+  rtensor = rtensor.contiguous(rtensor.suggest_memory_format());
```

Comment:
is this going to be a no-op?

Reply:
or this is still making the tensor contiguous?

Reply:
Ideally, it should become no-op.